src/io.rs

```diff
@@ -98,3 +100,82 @@ impl Write for CallbackWriter {
         Ok(())
     }
 }
+
+/// An alias for `struct iovec` from uio.h. You should cast `const struct rustls_iovec *` to
+/// `const struct iovec *`.
+#[cfg(unix)]
+pub struct rustls_iovec {
+    _private: [u8; 0],
+}
+
+/// A callback for rustls_connection_write_tls_vectored.
+/// An implementation of this callback should attempt to write the bytes in
+/// the given `count` rustls_slice_bytes to the network. If any bytes were written,
```

Comment:
This is confusing: there's a reference to `rustls_slice_bytes` but that type doesn't occur in the function signature. I assume `count` is the number of buffers in the `struct iovec`?